runtime/src/lib.rs

```diff
@@ -280,7 +280,7 @@ impl InstanceFilter<Call> for ProxyType {
             ProxyType::NonTransfer => !matches!(
                 c,
                 Call::Balances(..)
-                    | Call::Vesting(pallet_vesting::Call::vested_transfer(..))
+                    // | Call::Vesting(pallet_vesting::Call::vested_transfer(..))
```

Comment:
@Gauthamastro  Should maybe orml vesting connect to here somehow?

Reply:
Wait for my pr

Reply:
Good